unpackaged/config/trial/package.xml

```diff
@@ -131,7 +131,6 @@
         <members>LeadSource</members>
         <members>OpportunityStage</members>
         <members>OpportunityType</members>
-        <members>Salutation</members>
```

Comment:
Something is still not right here. The package.xml shouldn't be in the PR at all as changed in any way.